tests/Operation/FindAndModifyFunctionalTest.php

```diff
@@ -75,38 +74,16 @@ function (array $event) use ($expectedUpdate): void {
         );
     }
 
-    public function provideReplacementDocuments(): array
+    public function provideReplacementDocumentLikePipeline(): array
     {
-        $expected = (object) ['x' => 1];
-
-        return [
-            'replacement:array' => [['x' => 1], $expected],
-            'replacement:object' => [(object) ['x' => 1], $expected],
-            'replacement:Serializable' => [new BSONDocument(['x' => 1]), $expected],
-            'replacement:Document' => [Document::fromPHP(['x' => 1]), $expected],
-        ];
-    }
-
-    public function provideUpdateDocuments(): array
-    {
-        $expected = (object) ['$set' => (object) ['x' => 1]];
-
+        /* Note: this expected value differs from UpdateFunctionalTest because
+         * FindAndModify is not affected by libmongoc's pipeline detection for
+         * update commands (see: CDRIVER-4658). */
```

Comment:
I previously added this test case to UpdateFunctionalTest so we had a record of [CDRIVER-4658](https://jira.mongodb.org/browse/CDRIVER-4658). FindAndModify behaves correctly (unaffected by libmongoc's code path for `update` commands) and I wanted to make a record of that in the test suite.